addClass.php

```diff
@@ -14,17 +14,12 @@
 
             //$id = intval($_POST['id']);
             $className = $_POST['name'];
-            $volunteerId = intval($_POST['vid']);
-            $roomNum = $_POST['rnum'];
-            $srtTime = $_POST['stime'];
-            $endTime = $_POST['etime'];
-            $day = $_POST['day'];
 
 
 
 
-            $stmt = $db->prepare("INSERT INTO Class (Class_Name, Volunteer_Id) VALUES (?, ?)");
-            $stmt->bind_param('si',  $className, $volunteerId);
+            $stmt = $db->prepare("INSERT INTO Class (Class_Name) VALUES (?)");
+            $stmt->bind_param('si',  $className);
```

Comment:
im pretty sure you only need 's' because of our one parameter.  before it was a string and integer.  now we just have the string